e2etests/stable_test.go

```diff
@@ -12,6 +12,32 @@ var testMarkdown string
 
 func testStable(t *testing.T) {
 	tcs := []testCase{
+		{
+			name: "elk_border_radius",
+			script: `
+				a -> b
+				a -> c {
```

Comment:
this script is malformed. should be colons before maps

e.g.

```
a -> c: {
  ...
}
```

i guess it still works because we format it beforehand or something, but still the tests should use correct syntax

Reply:
fixed